scripts/pre_commit_linter.py

```diff
@@ -2489,6 +2489,57 @@ def _check_constants_declaration(self):
 
         return summary_messages
 
+    def _check_npm_packages(self):
+        """Checks to ensure that npm packages are not identified as vulnerable
+        by npm audit.
+        """
+        parent_dir = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+
+        npm_path = os.path.join(
+            parent_dir, 'oppia_tools', 'node-10.15.3', 'bin', 'npm')
+        if self.verbose_mode_enabled:
+            print 'Starting npm package vulnerability check...'
+            print '----------------------------------------'
+        print ''
+        if not self.verbose_mode_enabled:
+            print 'Checking npm packages.'
+
+        summary_messages = []
+        failed = False
+
+        proc_args = [npm_path, 'audit', '--parseable']
+        with _redirect_stdout(_TARGET_STDOUT):
+            proc = subprocess.Popen(
+                proc_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+            linter_stdout, _ = proc.communicate()
+
+            errors = linter_stdout.lstrip().rstrip()
+            if errors:
+                failed = True
+                error_list = errors.split('\n')
+                for line in error_list:
+                    line_tokens = line.split('\t')
+                    print ('%s is vulnerable. Please run "%s" to update.' %
```

Comment:
What does the error message look like here? Should we modify it to include the word "package", that is, `XXX package is vulnerable..`?

Reply:
Saw the error message just now. It looks good. I don't think we need to add the word "package". Resolving this conversation. Thanks!

Reply:
As for how the message looks like, please see a sample above. However, I've updated the message so it's like XXX package is vulnerable.